docs/install/java_setup.md

```diff
@@ -35,21 +35,15 @@ The following instructions are provided for macOS and Ubuntu. Windows is not yet
 brew update
 brew tap caskroom/versions
 brew cask install java8
-brew install opencv
 brew install maven
 ```
 
 **Ubuntu Steps**
 
-These scripts will install Maven and its dependencies. You will be running the Scala scripts because the MXNet-Java project has a dependency on the MXNet-Scala project.
+Please run the following lines:
 
 ```bash
-wget https://raw.githubusercontent.com/apache/incubator-mxnet/master/ci/docker/install/ubuntu_core.sh
```

Comment:
So all of the dependencies for ubuntu/scala/mxnet environment are covered with this change? I'm skeptical.

Reply:
We no longer require user to install that bunch... Simply Java and maven would work